### DISCUSSION TITLE
Standard installation for RPi4?

### DISCUSSION THREAD

**Discussion by pweigele** [opensteno/plover#1430] [Ideas]
As a lover of Plover, I'm wondering if there is any chance there could be a version of Plover that could be easily installed on Raspberry Pi v 3 or 4? I mean easy enough for a kid to do it.

**Comment by user202729** [comment]
This comment https://github.com/openstenoproject/plover/issues/1376#issuecomment-878381444 mostly implies that it should be possible, although I don't have any experience with this.

**Reply by pweigele** [reply]
Thanks, that thread is helpful. I keep hearing about the Wheel. Will try it out. Maybe one day, someone could write up an RPi specific installation section in the Plover docs on GitHub to save folks some time and also lower the barriers to entry for hobbyists looking to use the RPi hardware for learning steno.

**Reply by user202729** [reply]
I don't have such a hardware but I guess you can `pip install plover==4.0.0` or something similar. Didn't try however.
You do need to know how to use bash and pip.

**Comment by benoit-pierre** [comment]
I'm assuming your using the standard Raspberry Pi  OS. Because of a number of issues (outdated packages, missing Python metadata for the distribution provided PyQt5 package, no PyQt5 wheel in their custom PyPI index, ...), installation is a little complicated.

Try the [attached script](https://github.com/openstenoproject/plover/files/7449321/pi_install_plover.txt): download it and run it with `bash ./pi_install_plover.txt`.

**Reply by pweigele** [reply]
thank you so much for the script. it does look fairly complicated. but after a few tries, I think it did all the installations it needed to. so how do i run plover? do i have to call it from the command line or is there an icon-ized application installed in a directory somewhere?

**Reply by pweigele** [reply]
actually, no worries, i figured it out the last bit. just type plover at the command line! easy peasy.
why not put pi_install_plover.txt w short instructions on plover github?
